Detectors/ITSMFT/ITS/workflow/src/TrackerSpec.cxx

```diff
@@ -91,31 +67,34 @@ void TrackerDPL::run(ProcessingContext& pc)
   if (mState != 1)
     return;
 
-  auto compClusters = pc.inputs().get<const std::vector<o2::itsmft::CompClusterExt>>("compClusters");
-  auto clusters = pc.inputs().get<const std::vector<o2::itsmft::Cluster>>("clusters");
-  auto labels = pc.inputs().get<const o2::dataformats::MCTruthContainer<o2::MCCompLabel>*>("labels");
-  auto rofs = pc.inputs().get<const std::vector<o2::itsmft::ROFRecord>>("ROframes");
-  auto mc2rofs = pc.inputs().get<const std::vector<o2::itsmft::MC2ROFRecord>>("MC2ROframes");
+  auto compClusters = pc.inputs().get<const std::vector<itsmft::CompClusterExt>>("compClusters");
+  auto clusters = pc.inputs().get<const std::vector<itsmft::Cluster>>("clusters");
+  auto rofs = pc.inputs().get<const std::vector<itsmft::ROFRecord>>("ROframes");
 
   LOG(INFO) << "ITSTracker pulled " << clusters.size() << " clusters, "
-            << labels->getIndexedSize() << " MC label objects , in "
-            << rofs.size() << " RO frames and "
-            << mc2rofs.size() << " MC events";
+            << rofs.size() << " RO frames and ";
+
+  const dataformats::MCTruthContainer<MCCompLabel>* labels = mIsMC ? pc.inputs().get<const dataformats::MCTruthContainer<MCCompLabel>*>("labels").get() : nullptr;
```

Comment:
@mpuccio   This was quite a nice glitch.   Instead of "...("labels").get()" there should be "...("labels").release()".